### PR TITLE
Implement Vision servlet frontend

### PR DESCRIPTION
**Pull request by Froggerson**
image
image
image
image
Links to /CreateCard URL
image

### CODE REVIEW

File: frontend/src/main-components/ImageInterfaceContent.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  flex: 9;
+  display: flex;
+  border: 1rem solid white;
+  border-radius: 1rem;
+  background-color: white;
+  margin: 1%;
+  flex-direction: column;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.5rem;
+}
```
Comment (ESaracay) [inline frontend/src/main-components/ImageInterfaceContent.css]: Google style guide says to put a space between rules. https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules

File: frontend/src/sub-components/labelScroll.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  backgroundColor: "white",
+  color: "black",
+  borderRadius: "0.5rem",
+  fontSize: "1.75rem",
+};
+
+class LabelScroll extends Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (ESaracay) [inline frontend/src/sub-components/labelScroll.js]: You don't need this unless you're binding a function or setting a state.

File: frontend/src/sub-components/CropImage.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+const createImage = (url) =>
+	new Promise((resolve, reject) => {
+		const image = new Image();
+		image.addEventListener('load', () => resolve(image));
+		image.addEventListener('error', (error) => reject(error));
+		image.setAttribute('crossOrigin', 'anonymous');
+		image.src = url;
+	});
```
Comment (ESaracay) [inline frontend/src/sub-components/CropImage.js]: Something seems wrong with your spacing.

File: frontend/src/main-components/ImageInterfaceContent.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
+.Container {
```
Comment (tylerhou) [inline frontend/src/main-components/ImageInterfaceContent.css]: CSS styles should be camelcase and begin with a lowercase letter.

File: frontend/src/main-components/ImageInterfaceContent.js [hunk 5]
```diff
@@ -1,22 +1,213 @@
...
+          )}
+          {this.state.imageUploaded ? (
+            this.state.imageCropped ? (
+              <div className="ImageContainer">
+                <img
+                  className="CroppedImage"
+                  src={this.state.croppedImageURL}
+                />
+              </div>
+            ) : (
```
Comment (tylerhou) [inline frontend/src/main-components/ImageInterfaceContent.js:172]: Break this up into a separate component?

File: frontend/src/sub-components/CropImage.js (hunk 3 above)
Comment (tylerhou) [reply frontend/src/sub-components/CropImage.js]: Convert from tabs to spaces. Format with prettier and use `--tab-width 2`.

**Review by tylerhou — APPROVED** [review]

File: frontend/src/sub-components/labelScroll.js (hunk 2 above)
Comment (Froggerson) [reply frontend/src/sub-components/labelScroll.js]: alright!

**Pull request merged**